Makefile

```diff
@@ -11,3 +11,9 @@ build-alpine:
 	docker build -f Dockerfile-alpine -t jakzal/phpqa:alpine .
 .PHONY: build-alpine
 
+
+NIGHTLY_TAG := jakzal/phpqa:nightly-$(shell date +%y%m%d)
+build-nightly:
+	docker build -t $(NIGHTLY_TAG) .
+	@docker login -u ${DOCKERHUB_LOGIN} -p ${DOCKERHUB_PASSWORD}
```

Comment:
Is the login/pass the only way? No tokens?

Reply:
There's a way to trigger builds on docker hub afterall: #48 

Haven't tested it yet.

Reply:
That approach couldn't be used because I want new tag based on %currentday.